chia/seeder/start_crawler.py

```diff
@@ -23,36 +23,42 @@
 def service_kwargs_for_full_node_crawler(
     root_path: pathlib.Path, config: Dict, consensus_constants: ConsensusConstants
 ) -> Dict:
+    service_config = config[SERVICE_NAME]
+
     crawler = Crawler(
-        config,
+        service_config,
         root_path=root_path,
         consensus_constants=consensus_constants,
     )
     api = CrawlerAPI(crawler)
 
-    network_id = config["selected_network"]
+    network_id = service_config["selected_network"]
     kwargs = dict(
         root_path=root_path,
+        config=config,
         node=api.crawler,
         peer_api=api,
         node_type=NodeType.FULL_NODE,
-        advertised_port=config["port"],
+        advertised_port=service_config["port"],
         service_name=SERVICE_NAME,
         upnp_ports=[],
-        server_listen_ports=[config["port"]],
+        server_listen_ports=[service_config["port"]],
         on_connect_callback=crawler.on_connect,
         network_id=network_id,
     )
 
-    if config.get("crawler", {}).get("start_rpc_server", True):
-        kwargs["rpc_info"] = (CrawlerRpcApi, config.get("crawler", {}).get("rpc_port", 8561))
+    if service_config.get("crawler", {}).get("start_rpc_server", True):
+        kwargs["rpc_info"] = (CrawlerRpcApi, service_config.get("crawler", {}).get("rpc_port", 8561))
 
     return kwargs
 
 
 def main():
-    config = load_config_cli(DEFAULT_ROOT_PATH, "config.yaml", SERVICE_NAME)
-    overrides = config["network_overrides"]["constants"][config["selected_network"]]
+    # TODO: refactor to avoid the double load
+    config = load_config(DEFAULT_ROOT_PATH, "config.yaml")
+    service_config = load_config_cli(DEFAULT_ROOT_PATH, "config.yaml", SERVICE_NAME)
```

Comment:
```suggestion
    config = load_config_cli(DEFAULT_ROOT_PATH, "config.yaml")
    service_config = config[SERVICE_NAME]
```

Couldn't we just do ^ and then get rid of `load_config_cli` in a separete PR if its outdated and replace it here with `load_config`?

Reply:
This would change the behavior and the cli interface.  It would require the service name be included in the cli arguments and would apply the override behavior to the entire configuration.